### PR TITLE
meteor: add accounts-server token method typings

### PR DESCRIPTION
**Pull request by jnurkka**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/meteor/meteor/blob/0ef64807d30c3ff36b4a187124d0b548873e253b/packages/accounts-base/accounts_server.js#L964
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/meteor/accounts-base.d.ts [hunk 1]
```diff
@@ -236,4 +236,20 @@ declare module "meteor/accounts-base" {
...
+            when: Date;
+        }
+        type HashedStampedLoginToken = {
+            hashedToken: string;
+            when: Date;
+        }
+
+        function _generateStampedLoginToken(): StampedLoginToken;
+        function _hashStampedToken(token: StampedLoginToken): HashedStampedLoginToken;
+        function _insertHashedLoginToken(userId: string, token: HashedStampedLoginToken): void;
```
Comment (radekmie) [inline types/meteor/accounts-base.d.ts:252]: According to the [source](https://github.com/meteor/meteor/blob/6e61b5bc9ca8a8170b5ef28832010d0c12c2f5f2/packages/accounts-base/accounts_server.js#L830-L841), there's also third argument named `query` and I think it'd be good to add it.

**Review by radekmie — CHANGES_REQUESTED** [review]

File: types/meteor/accounts-base.d.ts (hunk 1 above)
Comment (jnurkka) [reply types/meteor/accounts-base.d.ts]: Thanks, [fixed](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50549/commits/f9ec1f94912836e722fb4d06b6db93aef997d202)

**Review by radekmie — APPROVED** [review]

**Comment by jnurkka** [comment]
Seems like the tests are flaky. CI / test check fails with some issues in `types/meteor-astronomy`, but when I run `npm test meteor-astronomy` locally, I don't get any errors. If I run `npm run test-all` locally, I get some errors with `types/react`..

**Comment by jnurkka** [comment]
@radekmie thanks for your review. nothing changed, I just squashed the commits into one, force-pushed and suddenly now all checks passed 🤷‍♂️

**Review by radekmie — APPROVED** [review]

**Comment by jnurkka** [comment]
Ready to merge

**Pull request merged**